lib/core/client.js

```diff
@@ -65,7 +65,7 @@ function getServerName (client, host) {
   return (
     util.getServerName(host) ||
     (client[kTLSOpts] && client[kTLSOpts].servername) ||
-    util.getServerName(client[kUrl].host || client[kUrl].hostname) ||
+    util.getServerName(client[kUrl].host /* istanbul ignore next */ || client[kUrl].hostname) ||
```

Comment:
This can happen. Please remove ignore.

Reply:
How comes? Since we are on Node LTS (so > 10 at the moment), `client[kUrl]` is a instance of `URL` (the WHATWG one), so if `host` is not defined, neither is `hostname`.

Am I wrong?

Reply:
you are right, then please remove the `||` part.

Reply:
Update coming soon!

Reply:
Done!